<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectVideoWindow</class>
 <widget class="QWidget" name="SelectVideoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>选择视频素材</string>
  </property>
  <property name="windowIcon">
   <iconset resource="orion2.qrc">
    <normaloff>:/images/unique32.png</normaloff>:/images/unique32.png</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>821</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="playpushButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="orion2.qrc">
        <normaloff>:/images/play.png</normaloff>:/images/play.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pausepushButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="orion2.qrc">
        <normaloff>:/images/pause.png</normaloff>:/images/pause.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="stoppushButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="orion2.qrc">
        <normaloff>:/images/stop.png</normaloff>:/images/stop.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>音量</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="volumehorizontalSlider">
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="volumelabel">
      <property name="minimumSize">
       <size>
        <width>24</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>24</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>50</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="loglabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="savepushButton">
      <property name="text">
       <string>确定</string>
      </property>
      <property name="icon">
       <iconset resource="orion2.qrc">
        <normaloff>:/images/ok.png</normaloff>:/images/ok.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelpushButton">
      <property name="text">
       <string>关闭</string>
      </property>
      <property name="icon">
       <iconset resource="orion2.qrc">
        <normaloff>:/images/cancel.png</normaloff>:/images/cancel.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>821</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QListWidget" name="listWidget">
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="VideoWidget" name="videowidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QWidget</extends>
   <header>videowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="orion2.qrc"/>
 </resources>
 <connections/>
</ui>
